tests/ignite/metrics/test_mean_squared_error.py

```diff
@@ -49,20 +49,59 @@ def update(engine, i):
             y_true[i * s + offset * rank : (i + 1) * s + offset * rank],
         )
 
-    engine = Engine(update)
+    def _test(metric_device):
+        engine = Engine(update)
 
-    m = MeanSquaredError()
-    m.attach(engine, "mse")
+        m = MeanSquaredError(device=metric_device)
```

Comment:
Tab used ?

Reply:
It just shows the diff with previous code, current one is indented and lint passes.

Reply:
Ok, strange rendering for me. I thought it was due to tab 😓